src/frontend/src/pages/index.js

```diff
@@ -42,66 +38,12 @@ export default function IndexPage() {
     getPosts(numPages);
   }, [telescopeUrl]);
 
-  async function getPostsCount() {
-    try {
-      const res = await fetch(`${telescopeUrl}/posts`, { method: 'HEAD' });
-      if (!res.ok) {
-        throw new Error(res.statusText);
-      }
-      return res.headers.get('x-total-count');
-    } catch (error) {
-      console.log(error);
-    }
-    return null;
-  }
-
-  function callback() {
-    getPostsCount()
-      .then(setCurrentNumPosts)
-      .catch((error) => console.log(error));
-  }
-
-  useEffect(() => {
-    savedCallback.current = callback;
-  });
-
-  // Get the current + initial posts count when page loads
-  useEffect(() => {
-    async function setPostsInfo() {
-      try {
-        await Promise.all([
-          setInitNumPosts(await getPostsCount()),
-          setCurrentNumPosts(await getPostsCount()),
-        ]);
-      } catch (error) {
-        console.log({ error });
-      }
-    }
-    setPostsInfo();
-  }, []);
-
-  useEffect(() => {
-    function getCurrentNumPosts() {
-      savedCallback.current();
-    }
-
-    savedCallback.current = callback;
-    // Polls every 5 minutes
-    const interval = setInterval(getCurrentNumPosts, 5 * 60 * 1000);
-    return () => clearInterval(interval);
-  }, [currentNumPosts]);
-
   return (
-    <PageBase>
-      <>
-        <Banner className="banner" />
-        <ScrollToTop />
-        <main className="main">
-          {posts.length > 0 ? <Posts posts={posts} /> : null}
-          {currentNumPosts !== initNumPosts ? <CustomizedSnackBar posts={currentNumPosts} /> : null}
-        </main>
-        <footer>© {new Date().getFullYear()}</footer>
-      </>
+    <PageBase title="Home">
+      <Banner />
+      <ScrollToTop />
+      <main>{posts.length > 0 ? <Posts posts={posts} /> : null}</main>
```

Comment:
change this line from `{posts.length > 0 ? <Posts posts={posts} /> : null}` to ` <Posts posts={posts} />` since we'll be doing the check in `Posts` component